pkg/pluginmanager/lifecycle.go

```diff
@@ -64,6 +65,7 @@ func start(steampipeExecutablePath string) (*State, error) {
 		Cmd:              pluginManagerCmd,
 		AllowedProtocols: []plugin.Protocol{plugin.ProtocolGRPC},
 		Logger:           logger,
+		StartTimeout:     3 * time.Minute, //TODO: macke this configurable
```

Comment:
The timeout for the plugin manager startup needs to be extended to accommodate longer plugin startup. I didn't see an immediate way to make this configurable, so I extended this from 1 minute to 3 minutes for now.